packages/backend/src/modules/tvl/TvlModule.ts

```diff
@@ -104,20 +105,20 @@ export function createTvlModule(
       logger,
     )
 
-  const submodules: (TvlSubmodule | undefined)[] = [
+  const submodules: TvlSubmodule[] = [
```

Comment:
Let's for now write a simple assert that will ensure all of them are defined when the env is production or staging

Reply:
Why would this be in the scope of this PR? I haven't changed the behaviour of this code.